From 915b99c9761672e6ec6d21b55c26f44c49c83c27 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Thu, 24 Jun 2010 15:58:43 +0000
Subject: Remove the last case of deprecated GTK+ api

By using gtk_container_child_get instead of
gtk_notebook_query_tab_label_packing. See bug 572178.
---
diff --git a/engines/clearlooks/src/support.c b/engines/clearlooks/src/support.c
index 05af83f..540720d 100644
--- a/engines/clearlooks/src/support.c
+++ b/engines/clearlooks/src/support.c
@@ -285,10 +285,10 @@ clearlooks_get_notebook_tab_position (GtkWidget *widget,
 				return;
 			}
 
-			gtk_notebook_query_tab_label_packing (notebook, tab_child,
-			                                      &expand,
-			                                      NULL, /* don't need fill */
-			                                      &pack_type);
+                        gtk_container_child_get (GTK_CONTAINER (notebook), tab_child,
+                                                 "expand", &expand,
+                                                 "pack-type", &pack_type,
+                                                 NULL);
 
 			if (!found_tabs) {
 				found_tabs = TRUE;
--
cgit v0.8.3.1
